test/test-time.c

```diff
@@ -104,6 +104,9 @@ main(int argc, char **argv)
 
 	for (i = 0; i < NEVENT; i++) {
 		ev[i] = evtimer_new(base, time_cb, event_self_cbarg());
+		if (ev[i] == NULL) {
```

Comment:
You need to add cleanup path, otherwise events allocated on the previous iterations will leak

Reply:
OK, I think I should also free the event_base.